src/ui/components/key_editor/ke_input.rs

```diff
@@ -1,9 +1,7 @@
 //! # Popups
 //!
-//!
 //! Popups components
```

Comment:
Can the doc comments like these be removed? I don't think they add much value, and they prevent the licenses from being at the top of the file.

Reply:
Yes, sure.